metaflow/plugins/kfp/kfp.py

```diff
@@ -435,9 +437,9 @@ def _create_resource_based_node_type_toleration(
         if memory >= memory_threshold or cpu >= cpu_threshold:
             return V1Toleration(
                 effect="NoSchedule",
-                key="node.kubernetes.io/instance-type",
+                key="node.k8s.zgtools.net/purpose",
                 operator="Equal",
-                value="r5.12xlarge",
+                value="high-memory",
```

Comment:
how about integration test change https://github.com/zillow/metaflow/blob/48750f5b32451cbb88abad253d5e75e28328b75e/metaflow/plugins/kfp/tests/run_integration_tests.py#L317-L317?

I did a search on `r5.12xlarge`

Reply:
Taleb you linked to version `48750f5b32451cbb88abad253d5e75e28328b75e`, but my commit version is `7d325380be7260ae2e0eaa2b9e3f0e14265a0fee` and is already changed to `high-memory`. See below: https://github.com/zillow/metaflow/blob/7d325380be7260ae2e0eaa2b9e3f0e14265a0fee/metaflow/plugins/kfp/tests/run_integration_tests.py#L317-L317

Reply:
Merging since this is a small change with less chance to go wrong, and test cases passed.